addons/mass_mailing/models/mass_mailing_stats.py

```diff
@@ -57,6 +57,7 @@ class MailMailStats(models.Model):
                                     help='Last state update of the mail',
                                     store=True)
     recipient = fields.Char(compute="_compute_recipient")
```

Comment:
Shoudn't we remove recipient while we are at it ?

Reply:
Addition as comment is not easyu to understand as it is: if we add email on statistics, do we have to keep recipient ?